### DISCUSSION TITLE
How to add blurred out Flags with Names

### DISCUSSION THREAD

**Discussion by ARandomRockN7** [axelboc/anki-ultimate-geography#650] [Q&A] [answered]
Hi all!! I'm trying to extend the note format so that I can use it as a universal baseline for all general geography cards. When attempting to work this in with using US states, how do I make it so that I can blur out the text that says the exact name of the flag? I saw that this was done with the "Guam" card, as shown in the included photos, so I was wondering how I would do this with other flags?
![Screenshot 2024-08-19 223822](https://github.com/user-attachments/assets/0cfc33d8-9a69-40ea-b1f1-eab2178bd37c)
![image](https://github.com/user-attachments/assets/6e5abb1a-0903-4de7-82f5-be8d17c690f4)
![image (1)](https://github.com/user-attachments/assets/487c871a-173b-4059-9ec1-4f10152fdbef)

**Comment by axelboc** [comment]
I believe this is documented in the [`CONTRIBUTING` guide](https://github.com/anki-geo/ultimate-geography/blob/master/CONTRIBUTING.md#flag-field) and the issue that is linked from there. If you get stuck, don't hesitate to get back to us :wink:

**Reply by ARandomRockN7** [reply]
> I believe this is documented in the [`CONTRIBUTING` guide](https://github.com/anki-geo/ultimate-geography/blob/master/CONTRIBUTING.md#flag-field) and the issue that is linked from there. If you get stuck, don't hesitate to get back to us 😉

I got the blur to start working, and it seems to work that the blurred flag is removed during the back card, but after following the information provided for formatting, it seems that during the front card, both cards are presented. Thoughts?
![Screenshot 2024-08-20 094148](https://github.com/user-attachments/assets/4de84d1c-86f1-4b84-b4e1-c2bc9e1c9141)
![Screenshot 2024-08-20 093959](https://github.com/user-attachments/assets/5dfd55ac-c34c-4702-b60d-d04d5b458979)
![ug-flag-arkansas-blur](https://github.com/user-attachments/assets/dcb45e69-64e6-4673-bf8e-78952444bb63)
![ug-flag-arkansas](https://github.com/user-attachments/assets/4e0f5886-cf9f-4aed-8d50-9a658144123c)

**Reply by aplaice** [reply, marked as answer]
Weird! I guess that the blurred flag is correctly named `...-blur.svg`? (Otherwise it wouldn't be hidden on the back side.)

What's the precise HTML code of the flag field for Arkansas? Is there a newline (e.g. `<br>`) (or actually anything else) between the two flags — if there is, remove it — the two images have to be direct "siblings" for [the CSS to work](https://github.com/anki-geo/ultimate-geography/blob/master/src/note_models/style.css#L64). (Just in case you weren't aware: you can view the HTML code in a field with Ctrl+Shift+x.) (This would be my best guess for what might be the problem...)

**Reply by ARandomRockN7** [reply]
Taking out the new line text break did the trick. Thank you!!

**Reply by aplaice** [reply]
I'm glad it worked!
